client/extensions/woocommerce/woocommerce-services/lib/pdf-label-utils/index.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ */
+import { translate as __ } from 'i18n-calypso';
+import querystring from 'querystring';
+import _ from 'lodash';
```

Comment:
Please use named import in order for the Tree Shaking to work properly ( aka don't load the whole lodash )